src/lib.rs

```diff
@@ -88,6 +88,37 @@
 //! At this time, we do not recommend use for security-critical applications.
 //! Use at your own risk.
 //!
+//! # 🔒 Networking
+//!
+//! The calling application is responsible for sending messages between
+//! participants in the protocol.
+//! All communication between [`Participant`]s must take place over a secure
+//! channel that satisfies mutual entity authentication and integrity.
```

Comment:
Nit: I would delete "secure" and say "... satisfies sender authentication and message integrity". Then delete L106 and move up L107-112.